test/jdk/sun/tools/jhsdb/JShellHeapDumpTest.java

```diff
@@ -150,7 +150,7 @@ public static void launchJshell() throws IOException {
         System.out.println("Starting Jshell");
         long startTime = System.currentTimeMillis();
         try {
-            ProcessBuilder pb = new ProcessBuilder(JDKToolFinder.getTestJDKTool("jshell"));
+            ProcessBuilder pb = new ProcessBuilder(JDKToolFinder.getTestJDKTool("jshell"), "-J-Duser.language=en", "-J-Duser.country=US");
```

Comment:
I just did a commit that changes this code. You'll need to rework your fix. Also, double check to make sure your changes are still necessary. I think they are, but it would be good to check first.

Reply:
Hello Chris,
Thank you for your review.

I understood that 8290497 has a similar fix, as you said.
JShellHeapDumpTest has not fixed yet.
The test always expects "Welcome to JShell" in English as jshell command message, but jshell prints localized message on the localized Windows platform.

I'm sorry, I didn't understand what rework I should do.
Could you give me advice?

I fixed the copyright year.

Thanks,
Kimura Yukihiro

Reply:
Hello Chris,
I realized that my fix conflicted to another fix.

Thanks,
Kimura Yukihiro

Reply:
> Hello Chris, I realized that my fix conflicted to another fix.

Yes, due to my changes you now have a merge conflict you'll need to resolve, and that will involve reworking your fix.